fortnite_api/stats.py

```diff
@@ -1,65 +1,284 @@
+        stats is a :attr:`fortnite_api.BrInputStats.squad` game mode or :attr:`fortnite_api.BrInputStats.overall`.
+    top5: Optional[:class:`int`]
+        The number of times the player has placed in the top 5 (10 players left remaining)
+        in this specific game mode. This is only available when the game mode
+        for these stats is a :attr:`fortnite_api.BrInputStats.duo` game mode or :attr:`fortnite_api.BrInputStats.overall`.
+    top6: Optional[:class:`int`]
+        The number of times the player has placed in the top 6 (25 players left remaining)
+        in this specific game mode. This is only available when the game mode
+        for these stats is a :attr:`fortnite_api.BrInputStats.squad` game mode or :attr:`fortnite_api.BrInputStats.overall`.
+    top10: Optional[:class:`int`]
+        The number of times the player has placed in the top 10 (10 players left remaining) in
+        this specific game mode. This is only available when the game mode for these
+        stats is a :attr:`fortnite_api.BrInputStats.solo` game mode or :attr:`fortnite_api.BrInputStats.overall`.
+    top12: Optional[:class:`int`]
+        The number of times the player has placed in the top 12 (25 players left remaining)
+        in this specific game mode. This is only available when the game mode
+        for these stats is a :attr:`fortnite_api.BrInputStats.duo` game mode or :attr:`fortnite_api.BrInputStats.overall`.
+    top25: Optional[:class:`int`]
+        The number of times the player has placed in the top 25 (25 players left remaining)
+        in this specific game mode. This is only available when the game mode for
+        these stats is a :attr:`fortnite_api.BrInputStats.solo` game mode or :attr:`fortnite_api.BrInputStats.overall`.
+    kills: :class:`int`
+        The total number of kills in this specific game mode.
+    kills_per_min: :class:`float`
+        The kills per minute for the stats for a specific game mode. This is the kills divided by the minutes played.
+    kills_per_match: :class:`float`
+        The kills per match for the stats for a specific game mode. This is the kills divided by the matches played.
+    deaths: :class:`int`
+        The total number of deaths in this specific game mode.
+    kd: :class:`float`
+        The kill/death ratio for this specific game mode. This is the kills divided by the deaths.
+    matches: :class:`int`
+        The total number of matches played in this specific game mode.
+    win_rate: :class:`float`
+        The win rate for this specific game mode. This is the wins divided by the matches played.
+    minutes_played: :class:`int`
+        The total number of minutes played in this specific game mode.
+    players_outlived: :class:`int`
+        The total number of players outlived in this specific game mode.
+    last_modified: :class:`datetime.datetime`
+        The date when this data was last modified.
```

Comment:
Is the `last_modified` field one that the Epic Games API provides, or is it a datetime that the Fortnite-API updates manually?

If the former, I agree completely - if the latter, however, it may be good to make a note that the field shows the last time the specific stat data was updated.

Reply:
> Is the last_modified field one that the Epic Games API provides

Yeah, it is. It shows the date, when the stats where updated within Epics API.